addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxHandler.java

```diff
@@ -57,6 +58,7 @@
 import org.openhab.binding.freebox.FreeboxBindingConstants;
 import org.openhab.binding.freebox.FreeboxDataListener;
 import org.openhab.binding.freebox.internal.config.FreeboxServerConfiguration;
+import org.openhab.binding.freebox.internal.discovery.FreeboxDiscoveryService;
```

Comment:
You should not introduce a dependency to the discovery service here, as the discovery service has already a dependency to the handler. As you only need it here for accessing the constants, it might make sense to move the constants here to the handler.